_build/build.sh

```diff
@@ -192,9 +192,11 @@ function build_pdf() {
   
   if [ "$SKIP_PDF_MANUAL" != "true" ]; then
     _build/buildPDF.sh "_site/zoneMergeManual.html" "_pdf/UserManual.pdf" 2>&1 | tee $PDF_MANUAL_LOG
+    _build/buildPDF.sh "_site/zoneMergeManual.html" "_pdf/UserManual_printable.pdf" "./style/css/printable.css" 2>&1 | tee $PDF_MANUAL_LOG
```

Comment:
not related to this PR but is there a reason this isn't `build-pdf.sh` consistent with the other commands in the dir?